applications/GeoMechanicsApplication/tests/cpp_tests/test_backward_euler_UPw_scheme.cpp

```diff
@@ -46,12 +47,9 @@ class BackwardEulerUPwSchemeTester
         p_node->AddDof(WATER_PRESSURE);
         result.GetProcessInfo()[DELTA_TIME] = 4.0;
 
-        p_node->FastGetSolutionStepValue(DISPLACEMENT, 1) =
-            Kratos::array_1d<double, 3>{7.0, 8.0, 9.0};
-        p_node->FastGetSolutionStepValue(VELOCITY, 1) =
-            Kratos::array_1d<double, 3>{1.0, 2.0, 3.0};
-        p_node->FastGetSolutionStepValue(ACCELERATION, 1) =
-            Kratos::array_1d<double, 3>{4.0, 5.0, 6.0};
+        p_node->FastGetSolutionStepValue(DISPLACEMENT, 1) = Kratos::array_1d<double, 3>{7.0, 8.0, 9.0};
+        p_node->FastGetSolutionStepValue(VELOCITY, 1) = Kratos::array_1d<double, 3>{1.0, 2.0, 3.0};
```

Comment:
automagic reformatting. Its a suprise that it doesn't align the = .

Reply:
It gives the same formatting for me, the problem is that the top line has 104 characters (more than our 100 char limit), so it won't align it